client/ayon_nuke/plugins/load/load_camera.py

```diff
@@ -196,3 +204,52 @@ def remove(self, container):
         node = container["node"]
         with viewer_update_and_undo_stop():
             nuke.delete(node)
+
+
+class AlembicCameraLoader(_NukeCameraLoader):
+    """
+    This will load alembic camera into script.
+    """
+    extensions = {"abc"}
+    label = "Load Alembic Camera"
+    enabled = True
+
+
+class FbxCameraLoader(_NukeCameraLoader):
+    """
+    This will load fbx camera into script.
+    """
+    extensions = {"fbx"}
+    label = "Load FBX Camera"
+    enabled = True
+
+    def load(self, context, name, namespace, data):
+        fbx_camera_node = super().load(context, name, namespace,data)
+
+        # Nuke forces 7 standard FBX cameras
+        # Producer Perspective, Producer Top, Producer Bottom...
+        # https://learn.foundry.com/nuke/11.1/content/comp_environment
+        # /3d_compositing/importing_fbx_cameras.html
+        # The following ensure the camera node is set to exported camera data.
+        fbx_camera_node.forceValidate() 
+
+        try:
+            knob = fbx_camera_node["fbx_take_name"]
+            knob.setValue(0)
+
+        except NameError:
+            # WARNING - Nuke 15: Camera3 validate does not play along
+            # very well when mixing abc + fbx nodes in the same script.
+            # They need to reloaded manually.
+            #
+            # Hopefully this will be fixed by upcoming Camera4            
+            raise RuntimeError(
+                "Could not load incoming FBX camera properly. "
+                "This could be caused by a mix of abc and fbx "
+                "into the script."
+            )
```

Comment:
You can reproduce this by trying to load a mix of Camera from published ABC and FBX files:
1. Load an ABC camera
2. Append a FBX camera to the same Nuke scene
3. FBX Camera node is not properly loaded 

After some testing I feel this is coming from Nuke trying to be smart and re-using the **fbx_node_name** knob.

for Alembic
![image](https://github.com/user-attachments/assets/8426d43a-7afc-44a4-870f-1392e789e5f7)


for FBX
![image](https://github.com/user-attachments/assets/e29082fa-8538-47fd-a1ad-c1a40dc9515a)